API.md

```diff
@@ -31,7 +31,7 @@ The `'cookie`' scheme takes the following options:
   current session for a new `ttl` duration. Defaults to `false`.
 - `redirectTo` - optional login URI or function `function(request)` that returns a URI to redirect unauthenticated requests to. Note that it will only
   trigger when the authentication mode is `'required'`. To enable or disable redirections for a specific route,
-  set the route `plugins` config (`{ options: { plugins: { 'hapi-auth-cookie': { redirectTo: false } } } }`).
+  set the route `plugins` config (`{ options: { plugins: { cookie: { redirectTo: false } } } }`).
```

Comment:
I'm not really keen on this change. Indeed this is the default/official plugin to work with cookies auth in the hapijs ecosystem but still I wonder if we shouldn't keep the scope here. We could have `@hapijs/cookie` instead of `cookie`.

Reply:
I get your point, but for instance [crumb](https://hapi.dev/module/crumb/) uses `crumb` (but not `@hapi/crumb`) to define route-level options. I think it makes more sense to keep it simple/short and to do it the way it has already been done.

Reply:
That is a fair point.  And when you use [`server.expose()`](https://hapi.dev/api/?v=20.1.0#-serverexposekey-value-options) it also (by default but configurable) drops the `@hapi/` when placing the value on `server.plugins`, so there's somewhat of a precedent for this within hapi too.  E.g.

```js
// Inside @hapi/biscuit (not a real plugin)
server.expose('butteriness', true);

expect(server.plugins.biscuit.butteriness).to.equal(true);
```